build/Targets/VSL.Versions.targets

```diff
@@ -72,13 +72,10 @@
     </BuildNumberPart2>
 
     <!-- Only set when building RTM with no dependencies on pre-release packages
```

Comment:
This can now be simplified. We can just have a single NuGetVersion variable. Then in setup.proj file we can remove 2 out of the 3 calls to BuildNuGet.csx